docs/detections/detections-exclude-cold-frozen-data-tiers.asciidoc

```diff
@@ -1,18 +1,24 @@
 [[exclude-cold-frozen-data-individual-rules]]
-== Exclude cold and frozen data from a rule
+== Exclude cold and frozen data from rule executions
```

Comment:
I think the title should be updated to make it more clear and explicit that this workflow applies to **SINGLE** rules, not **ALL** rules.

Reply:
@peluja1012 how's this:

```suggestion
== Exclude cold and frozen data from rule executions for individual rules
```

Reply:
@marshallmain @peluja1012 how's this:

```suggestion
== Exclude cold and frozen data from individual rules
```

Reply:
Thanks! look good to me.